NickvisionMoney.Shared/Resources/Strings.nl.resx

```diff
@@ -0,0 +1,677 @@
+		<value>Nieuwe rekening</value>
+	</data>
+
+	<data name="NewAccount.Tooltip" xml:space="preserve">
+		<value>Nieuwe rekening (Ctrl+N)</value>
+	</data>
+
+	<data name="UnableToOverride" xml:space="preserve">
+		<value>Een rekening kan slechts eenmaal worden geopend.</value>
+	</data>
+
+	<data name="OpenAccount" xml:space="preserve">
+		<value>Rekening tonen</value>
+	</data>
+
+	<data name="OpenAccount.Tooltip" xml:space="preserve">
+		<value>Rekening tonen (Ctrl+O)</value>
+	</data>
+
+	<data name="NoAccountOpened" xml:space="preserve">
+		<value>Er is geen rekening geopend</value>
+	</data>
+
+	<data name="NoAccountDescription" xml:space="preserve">
+		<value>Open een rekening of voeg er een toe om aan de slag te gaan. Ook kunt u een rekeningbestand hierheen slepen.</value>
+	</data>
+
+	<data name="RecentAccounts" xml:space="preserve">
+		<value>Recent Accounts</value>
+	</data>
+
+	<data name="NoRecentAccounts" xml:space="preserve">
+		<value>Geen onlangs gebruikte rekeningen</value>
+	</data>
+
+	<data name="AllFiles" xml:space="preserve">
+		<value>Alle bestanden</value>
+	</data>
+
+	<!--MainWindowController-->
+	<data name="Greeting.Generic" xml:space="preserve">
+		<!-- Shown when system time can't be detected -->
+		<value>Goedendag!</value>
+	</data>
+
+	<data name="Greeting.Morning" xml:space="preserve">
+		<!-- Shown from 6am to noon -->
+		<value>Goedemorgen!</value>
+	</data>
+
+	<data name="Greeting.Afternoon" xml:space="preserve">
+		<!-- Shown from noon to 6pm -->
+		<value>Goedemiddag!</value>
+	</data>
+
+	<data name="Greeting.Evening" xml:space="preserve">
+		<!-- Shown from 6pm to mignight -->
+		<value>Goedenavond!</value>
+	</data>
+
+	<data name="Greeting.Night" xml:space="preserve">
+		<!-- Shown from midnight to 6am -->
+		<!-- IMPORTANT: do not just translate literally, use whatever is natural for your language -->
+		<value>Goedenacht!</value>
+	</data>
+
+	<data name="AccountOpenedAlready" xml:space="preserve">
+		<value>Deze rekening wordt al getoond.</value>
+	</data>
+
+	<!--AccountView-->
+	<data name="NewTransaction" xml:space="preserve">
+		<value>Nieuwe overboeking</value>
+	</data>
+
+	<data name="NewTransaction.Short" xml:space="preserve">
+		<value>Nieuw</value>
+	</data>
+
+	<data name="NewTransaction.Tooltip" xml:space="preserve">
+		<value>Nieuwe overboeking (Ctrl+N)</value>
+	</data>
+
+	<data name="NewGroup" xml:space="preserve">
+		<value>Nieuwe groep</value>
+	</data>
+
+	<data name="NewGroup.Short" xml:space="preserve">
+		<value>Nieuw</value>
+	</data>
+
+	<data name="NewGroup.Tooltip" xml:space="preserve">
+		<value>Nieuwe groep (Ctrl+Shift+N)</value>
+	</data>
+
+	<data name="TransferMoney" xml:space="preserve">
+		<value>Geld overmaken</value>
+	</data>
+
+	<data name="TransferMoney.Tooltip" xml:space="preserve">
+		<value>Geld overmaken (Ctrl+T)</value>
+	</data>
+
+	<data name="ImportFromFile" xml:space="preserve">
+		<value>Import from File</value>
```

Comment:
Not translated

Reply:
Odd. I guess my text editor didn't save the file after my last edits…